### PR TITLE
Decommission DevParametersHttpRequestHandler

### PR DESCRIPTION
**Pull request by shtukas**
## What does this change?

Removes obsolete variables `httpRequestHandler`, leading to the decommission of `DevParametersHttpRequestHandler`.

### CODE REVIEW

**Comment by shtukas** [comment]
Will be properly tested on CODE

File: applications/app/AppLoader.scala [hunk 1]
```diff
@@ -10,14 +10,14 @@ import conf.CachedHealthCheckLifeCycle
...
+import dev.DevAssetsController
 import http.{CommonFilters, CorsHttpErrorHandler}
 import jobs.{SiteMapJob, SiteMapLifecycle}
-import model.{ApplicationContext, ApplicationIdentity}
+import model.ApplicationIdentity
 import services.ophan.SurgingContentAgentLifecycle
 import play.api.ApplicationLoader.Context
 import play.api.BuiltInComponentsFromContext
-import play.api.http.{HttpErrorHandler, HttpRequestHandler}
+import play.api.http.HttpErrorHandler
```
Comment (mxdvl) [inline applications/app/AppLoader.scala:20]: Slight inconsistency with the other files:

```suggestion
import play.api.http.{HttpErrorHandler}
```

**Review by mxdvl — COMMENTED** [review]
What do we mean by “unused”? It would be helpful to have a little bit more context.

**Comment by shtukas** [comment]
@mxdvl It means that it's dead code 🙂 Nothing is actually using that class so it can be removed. In normal circumstances, I would not care, but it prevents compilation in Play 2.7, so I am removing it.

**Pull request merged**